src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((chatEntry, i) => {
+    return <ChatEntry
+    key={i}
+    id={chatEntry.id}
+    sender={chatEntry.sender}
+    body={chatEntry.body}
+    timeStamp={chatEntry.timeStamp}
+    liked={chatEntry.liked}
+    onUpdateChatData={props.onUpdateChatData}
+    />
+  })
+  
+  return (
+  <div>
+    {chatEntries}
+  </div>
+  )
+};
```

Comment:
👍  Great work setting up the `Chatlog` component.

Reply:
One thing that we can do to DRY up this component is by using object restructuring to define `props` and remove the constant need to use dot notation with `chatEntry.nameOfprop`. 

```javascript
const ChatLog = ({ id, sender, body, timeStamp, liked, onUpdateChatData}) => {
  const chatEntries = props.entries.map((chatEntry, i) => {
    return <ChatEntry
    key={i}
    id={id}
    sender={sender}
    body={body}
    timeStamp={timeStamp}
    liked={liked}
    onUpdateChatData={onUpdateChatData}
    />
  })
  
  return (
  <div>
    {chatEntries}
  </div>
  )
};
```